styles/signup.css

```diff
@@ -0,0 +1,63 @@
+.search_navbar {
+  padding: 10px;
+  width: var(--width-100);
+}
+.cake-logo {
+  color: var(--color-white);
+  font-size: var(--size-7);
+}
+.fa-search {
+  position: absolute;
+  top: 43%;
+  left: 21%;
+  cursor: pointer;
+}
+.list {
+  justify-content: space-around;
+}
+.list_end {
+  padding-left: 10%;
+}
+.footer {
```

Comment:
If you are using same footer code in multiple files, then simply create one single `footer.css` and import it in each file.